diff --git a/cmd/service/main.go b/cmd/service/main.go
index 94084d8..3390c1a 100644
--- a/cmd/service/main.go
+++ b/cmd/service/main.go
@@ -22,22 +22,11 @@ import (
 	"time"
 )
 
-const snapSetupDoneFilePath = "/var/snap/wifi-ap/common/.setup_done"
-
 func main() {
 	s := &service{}
 
-	// Wait until the configure hook, which is called on snap
-	// installation, marked us as successfully setup. If we
-	// continue before that happen we will miss any initial
-	// configuration set via a gadget snap.
-	for {
-		_, err := os.Stat(snapSetupDoneFilePath)
-		if err == nil {
-			break
-		}
-		time.Sleep(time.Second/2)
-	}
+	// Sleep 10 seconds to get system to settle before starting AP
+	time.Sleep(10*time.Second)
 
 	c := make(chan os.Signal, 1)
 	signal.Notify(c, syscall.SIGHUP, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT)
